manifests/centosimage.pp

```diff
@@ -29,18 +30,32 @@
     before  => Docker::Image['centosagent']
   }
 
+  # Mount the gem cache directory because docker doesn't follow symlinks
+  # mount --bind /var/cache/rubygems rubygems
+  file {'/var/docker/centosagent/rubygems':
+    ensure => directory,
+    before => Mount['/var/docker/centosagent/rubygems'],
+  }
+
+  mount {'/var/docker/centosagent/rubygems':
```

Comment:
Do we need the gems at docker image build time @greglarkin? If not you could just mount the directory when the container is started instead of doing it here. I actually want to remove the other mount because it makes the container bigger and then the packages are static